wgsl/index.bs

```diff
@@ -1867,17 +1940,17 @@ Variables at [=module scope=] are restricted as follows:
     have a storage class decoration.  The storage class will always be [=storage classes/handle=].
 
 A variable in the [=storage classes/uniform=] storage class is a <dfn noexport>uniform buffer</dfn> variable.
-Its [=store type=] must be a [=host-shareable=] structure type with [=block=] attribute,
+Its [=store type=] must be a [=host-shareable=] structure type with [=attribute/block=] attribute,
 satisfying the (storage class constraints)[#storage-class-constraints].
 
 A variable in the [=storage classes/storage=] storage class is a <dfn noexport>storage buffer</dfn> variable.
-Its [=store type=] must be a [=host-shareable=] structure type with [=block=] attribute,
+Its [=store type=] must be a [=host-shareable=] structure type with [=attribute/block=] attribute,
 satisfying the (storage class constraints)[#storage-class-constraints].
```

Comment:
Looks like these don't render properly either.  Ooops.  I can fix them all after landing.

Reply:
I fixed this while resolving the merge conflict with upstream.